<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1425</width>
    <height>1089</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
      <item>
       <widget class="QTabWidget" name="twMain">
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tabCrimes">
         <attribute name="title">
          <string>Правонарушения</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tabPersons">
         <attribute name="title">
          <string>Собственники ТС</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="childrenCollapsible">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QSplitter" name="splitter">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <widget class="QWidget" name="wtOwners" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>Собственники ТС</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leSearchOwner">
                      <property name="placeholderText">
                       <string>Введите данные для начала поиска ...</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QTableView" name="tvOwners">
                      <property name="horizontalScrollBarPolicy">
                       <enum>Qt::ScrollBarAlwaysOff</enum>
                      </property>
                      <property name="sizeAdjustPolicy">
                       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
                      </property>
                      <property name="editTriggers">
                       <set>QAbstractItemView::NoEditTriggers</set>
                      </property>
                      <property name="alternatingRowColors">
                       <bool>true</bool>
                      </property>
                      <property name="selectionMode">
                       <enum>QAbstractItemView::SingleSelection</enum>
                      </property>
                      <property name="selectionBehavior">
                       <enum>QAbstractItemView::SelectRows</enum>
                      </property>
                      <property name="sortingEnabled">
                       <bool>true</bool>
                      </property>
                      <property name="cornerButtonEnabled">
                       <bool>false</bool>
                      </property>
                      <attribute name="horizontalHeaderCascadingSectionResizes">
                       <bool>true</bool>
                      </attribute>
                      <attribute name="horizontalHeaderHighlightSections">
                       <bool>false</bool>
                      </attribute>
                      <attribute name="horizontalHeaderStretchLastSection">
                       <bool>true</bool>
                      </attribute>
                      <attribute name="verticalHeaderVisible">
                       <bool>false</bool>
                      </attribute>
                     </widget>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_2">
                      <item>
                       <spacer name="horizontalSpacer_2">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>248</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="QPushButton" name="btnAddOwner">
                        <property name="minimumSize">
                         <size>
                          <width>100</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Добавить</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="btnUpdateOwner">
                        <property name="minimumSize">
                         <size>
                          <width>100</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Изменить</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="btnRemoveOwner">
                        <property name="minimumSize">
                         <size>
                          <width>100</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Удалить</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="wtVehicles" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
                      <item>
                       <widget class="QLabel" name="label_2">
                        <property name="text">
                         <string>Транспортные средства:</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="leOwnerName">
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="QTableView" name="tvVehicles">
                      <property name="horizontalScrollBarPolicy">
                       <enum>Qt::ScrollBarAlwaysOff</enum>
                      </property>
                      <property name="editTriggers">
                       <set>QAbstractItemView::NoEditTriggers</set>
                      </property>
                      <property name="alternatingRowColors">
                       <bool>true</bool>
                      </property>
                      <property name="selectionMode">
                       <enum>QAbstractItemView::SingleSelection</enum>
                      </property>
                      <property name="selectionBehavior">
                       <enum>QAbstractItemView::SelectRows</enum>
                      </property>
                      <property name="sortingEnabled">
                       <bool>true</bool>
                      </property>
                      <attribute name="horizontalHeaderStretchLastSection">
                       <bool>true</bool>
                      </attribute>
                      <attribute name="verticalHeaderVisible">
                       <bool>false</bool>
                      </attribute>
                     </widget>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_5">
                      <item>
                       <spacer name="horizontalSpacer_3">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>248</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="QPushButton" name="btnAddVehicle">
                        <property name="minimumSize">
                         <size>
                          <width>100</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Добавить</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="btnUpdateVehicle">
                        <property name="minimumSize">
                         <size>
                          <width>100</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Изменить</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="btnRemoveVehicle">
                        <property name="minimumSize">
                         <size>
                          <width>100</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Удалить</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="wtCrimes" native="true"/>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabVehicles">
         <attribute name="title">
          <string> Транспортные средства</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>998</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnCloseApp">
          <property name="text">
           <string>Выход</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCloseApp</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1113</x>
     <y>934</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
